website/manual.md

```diff
@@ -120,6 +120,12 @@ Using [Homebrew](https://brew.sh/) (mac):
 brew install deno
 ```
 
+To compile from source sit down with a good book and run:
```

Comment:
😅 it's not that bad, is it? It takes roughly 5-7 mins to compile from scratch on my machine

Reply:
fixed text